src/wp-content/themes/twentyfifteen/css/blocks.css

```
@@ -123,6 +123,11 @@ p.has-drop-cap:not(:focus)::first-letter {
 	text-transform: none;
 }
 
+.wp-block-quote[class*="-text-color"] cite,
+.wp-block-quote[style*="color"] cite {
```

```suggestion
.wp-block-quote.has-text-color cite,
.wp-block-quote.has-background cite {
```

For colors, please use specific class names such as `.has-text-color` and/or `.has-background`. The loose comparison for `[style*="color"]` can match text, background and/or border color properties.

---

Added Suggested changes.